packages/storage-driver-s3/src/index.ts

```diff
@@ -194,6 +197,10 @@ export class DriverS3 implements TusDriver {
 
 		if (this.config.serverSideEncryption) {
 			params.ServerSideEncryption = this.config.serverSideEncryption;
+
+			if (this.config.serverSideEncryption === 'aws:kms' && this.config.serverSideEncryptionKmsKeyId) {
+				params.SSEKMSKeyId = this.config.serverSideEncryptionKmsKeyId;
+			}
```

Comment:
I think this check should be added for `createChunkedUpload` as it's also another upload command